lib/api/redfish-1.0/systems.js

```diff
@@ -280,7 +282,13 @@ var getSystem = controller(function(req, res) {
             }).catch(function(error) {
                 return redfish.handleError(error, res);
             });
-        } else {
+        } else if(result.vendor === 'Cisco'){
+	     return dataFactory(identifier,'UCS').then(function(data){
```

Comment:
You should get data from UCS:blade-x for blade servers and UCS for rack mount servers.